src/templates/login-full.html

```
@@ -3,13 +3,11 @@
 </div>
 
 <div class='text-center'>
-  <div class='col-md-3 mx-auto'>
-    <div class='logo logo-light'>
-      {{ App.Config.configArr.logo_light_svg|raw }}
-    </div>
-    <div class='logo logo-dark'>
-      {{ App.Config.configArr.logo_dark_svg|raw }}
-    </div>
+  <div class='logo logo-light'>
+     <img src='/api/v2/instance/branding/{{ constant('Elabftw\\Enums\\Branding::Light').value }}?format=binary' alt='eLabFTW'>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)